examples/cactus-example-carbon-accounting-backend/package.json

```diff
@@ -77,12 +77,15 @@
     "@types/express": "4.17.19",
     "@types/fs-extra": "9.0.13",
     "@types/json-stable-stringify": "1.0.34",
+    "@types/qs": "^6",
```

Comment:
@aldousalvarez Please specify the exact version so that it is pinned down.

Reply:
apologies on this one @petermetz , it is now fixed and I also forgot to add yarn config set nodeLinker node-modules on the dockerfile

Reply:
@aldousalvarez Great, thank you!